src/main/java/fr/nocsy/mcpets/MCPets.java

```diff
@@ -132,6 +135,24 @@ private static void checkLuckPerms() {
         }
     }
 
+    private static boolean checkItemsAdder() {
+        if (itemsAdder != null) {
+            return true;
+        }
+    
+        if (Bukkit.getPluginManager().getPlugin("ItemsAdder") != null) {
+            try {
+                Class<?> customStackClass = Class.forName("dev.lone.itemsadder.api.CustomStack");
+                itemsAdder = customStackClass;
+                return true;
+            } catch (ClassNotFoundException e) {
+                Bukkit.getLogger().warning("[MCPets] : ItemsAdder API not found.");
```

Comment:
Making it a warning is probably deceiving for people, I've had complains about it previously for the MySQL that I need to change too. I would use INFO and probably add what features won't be available (so people know it's not mandatory).